Tasks/Common/webdeployment-common/msdeployutility.ts

```diff
@@ -101,21 +99,36 @@ export function getMSDeployCmdArgs(webAppPackage: string, webAppName: string, pu
  */
 export async  function containsParamFile(webAppPackage: string ) {
```

Comment:
The function does more than checking presence of param file now. It also clones param file if present. Change the name to reflect the behavior.